src/applications/apps.ts

```diff
@@ -373,7 +401,7 @@ export function validateRegisterWithConfig(config) {
     );
 }
 
-function validCustomProps(customProps) {
+function validCustomProps(customProps: any): boolean {
```

Comment:
It may be reasonable to use `unknown` instead of `any` here.

Reply:
changed